drivers/usb/device/usb_dc_stm32.c

```diff
@@ -89,6 +89,13 @@ static const struct gpio_dt_spec ulpi_reset =
 	GPIO_DT_SPEC_GET_OR(DT_PHANDLE(DT_INST(0, st_stm32_otghs), phys), reset_gpios, {0});
 #endif
 
```

Comment:
Nits: a comment above this block to explain where these are used 
```suggestion
/**
 * The following defines are used to map the value of the "maxiumum-speed"
 * DT property to the corresponding definition used by the STM32 HAL.
 */
```